packages/build/src/tracing/main.ts

```diff
@@ -29,8 +38,32 @@ const getOtelLogger = function (logger: (...args: any[]) => void): DiagLogger {
   }
 }
 
+/**
+ * Gets the global context to be used when initialising our root span
+ * TODO this will move to a shared package (opentelemetry-utils) to scope the usage of this global property there
+ */
+export const getGlobalContext = function (): Context {
+  if (global['NETLIFY_GLOBAL_CONTEXT'] === undefined) {
+    return context.active()
+  }
+  return global['NETLIFY_GLOBAL_CONTEXT']
+}
+
+/**
+ * Sets global context to be used when initialising our root span
+ * TODO this will move to a shared package (opentelemetry-utils) to scope the usage of this global property there
+ */
+export const setGlobalContext = function (ctx: Context) {
+  global['NETLIFY_GLOBAL_CONTEXT'] = ctx
+}
+
```

Comment:
This has been pulled from - https://github.com/netlify/build/blob/21239dca368001072a3cead8805a3c641f744fd5/packages/opentelemetry-sdk-setup/src/util.ts#L105-L122 - the idea is to move this to shared package as a follow up.